other/ethhub-cftc-response.md

```diff
@@ -10,11 +10,16 @@ DEADLINE: February 9, 2019
 
 ### 1. What was the impetus for developing Ether and the Ethereum Network, especially relative to Bitcoin?
 
-It's first vitally important to distinguish between Ether and Ethereum. Ethereum is an open-source, blockchain-based distributed ledger with decentralized control that supports general computing and economic activities. Anyone is able to build and deploy decentralized products and services that run on top of Ethereum by creating highly flexible programmable transactions, called 'smart contracts'. This smart contract technology has proven to be attractive to developers since it enables them to create programs and business logic that run exactly as coded, trustlessly and with no down time.
+The underlying impetus to develop Ethereum was to utilize aspects of the technology initially developed as part of the Bitcoin blockchain and combine it with the capabilities of a Turing complete supporting environment.
```

Comment:
Turing complete is useful here. Might need to define Turing complete for the laymen though.

Reply:
I saw your sources down below. I think it is still use to give an overview of Turing completeness in addition to providing your sources.